freqtrade/rpc/telegram.py

```diff
@@ -517,6 +533,7 @@ def _help(self, bot: Bot, update: Update) -> None:
                   "*/reload_conf:* `Reload configuration file` \n" \
                   "*/whitelist:* `Show current whitelist` \n" \
                   "*/blacklist [pair]:* `Show current blacklist, or adds one or more pairs " \
+                  "*/edge:* `Shows validated pairs by Edge if it is enabeld` \n" \
```

Comment:
Please move this one line lower ... you're in the middle of the  "blacklist"  output, which spans 2 lines (flake8 ...)